DEVELOPMENT.md

```diff
@@ -1,72 +1,167 @@
-## Install BentoML from source
+# Developer Guide
 
-Make sure to have [Git](https://git-scm.com/) and [Python3.7+](https://www.python.org/downloads/) installed.
+## Start Developing
 
-Optionally, make sure to have [GNU Make](https://www.gnu.org/software/make/) available on your system if you aren't using UNIX-based system for better developer experience.
-If you don't want to use `make` then refers to [Makefile](./Makefile) for specific commands on given make target.
+<details><summary><h3>with the Command Line</h3></summary>
 
-```bash
-python --version
+1. Make sure to have [Git](https://git-scm.com/), [pip](https://pip.pypa.io/en/stable/installation/), and [Python3.7+](https://www.python.org/downloads/) installed.
 
-pip --version
-```
+   Optionally, make sure to have [GNU Make](https://www.gnu.org/software/make/) available on your system if you aren't using a UNIX-based system for a better developer experience.
+   If you don't want to use `make` then please refer to the [Makefile](./Makefile) for specific commands on a given make target.
 
-Clone the source code from BentoML's GitHub repository:
-```bash
-git clone https://github.com/bentoml/BentoML.git && cd BentoML
-```
+2. Fork the BentoML project on [GitHub](https://github.com/bentoml/BentoML).
 
-Install BentoML with pip in `editable` mode:
-```bash
-pip install -e .
-```
+3. Clone the source code from your fork of BentoML's GitHub repository:
+   ```bash
+   git clone https://github.com/yourusername/BentoML.git && cd BentoML
```

Comment:
Actually we want to keep this like before, since this sections tells users how to install bentoml from the official repository, below this section we do mention how to install from fork.

Reply:
Since it's a guide for developers I think the correct thing to do is to tell them to fork. I don't think it really makes sense for somebody who doesn't know what they're doing to clone the master repo, and if you do know what you're doing you probably don't need the help.